fireworks_poe_bot/fw_poe_server_bot.py

```diff
@@ -30,12 +74,16 @@ def __init__(
         environment: str,
         server_version: str,
         completion_async_method: Callable = ChatCompletion.acreate,
+        allow_attachments: bool = False,
     ):
         super().__init__()
         self.model = model
         self.environment = environment
         self.server_version = server_version
         self.completion_async_method = completion_async_method
+        self.allow_attachments = allow_attachments
+        logging_level = getattr(logging, os.environ.get("LOGGING_LEVEL", "INFO"))
+        logging.basicConfig(level=logging_level)
```

Comment:
Actually can you copy this file over and import it? https://github.com/fw-ai/fireworks_poe_qr_bot/blob/main/fireworks_poe_qr_bot/logging.py